rust/main/agents/relayer/src/msg/op_batch.rs

```diff
@@ -43,26 +43,11 @@ impl OperationBatch {
             }
         };
 
-        if let Some(first_item) = excluded_ops.first() {
-            let Some(mailbox) = first_item.try_get_mailbox() else {
-                error!(excluded_ops=?excluded_ops, "Excluded ops don't have mailbox while they should have");
-                return; // we expect that excluded ops have mailbox
-            };
-
-            if mailbox.supports_batching() {
-                warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Sending them back to prepare queue.");
-                let reason = ReprepareReason::ErrorEstimatingGas;
-                let status = Some(PendingOperationStatus::Retry(reason.clone()));
-                for mut op in excluded_ops.into_iter() {
-                    op.on_reprepare(None, reason.clone());
-                    prepare_queue.push(op, status.clone()).await;
-                }
-            } else {
-                info!(excluded_ops=?excluded_ops, "Chain does not support batching. Submitting serially.");
-                OperationBatch::new(excluded_ops, self.domain)
-                    .submit_serially(prepare_queue, confirm_queue, metrics)
-                    .await;
-            }
+        if !excluded_ops.is_empty() {
+            warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Submitting them serially.");
```

Comment:
let's reuse the original warning message for consistent log filtering:
```suggestion
            warn!(excluded_ops=?excluded_ops, "Either operations reverted in the batch or the txid wasn't included. Falling back to serial submission.");
```